ts/process_compose.ts

```diff
@@ -30,12 +31,5 @@ export const processCompose = (executables: Record<string, Executable>) => {
     nixRaw`pkgs.writeText "process-compose.yml" (builtins.toJSON ${processComposeConfig})`
   );
 
-  return mkProject(
-    {
-      description: "",
-      defaultEnvironment: emptyEnvironment,
-      defaultExecutable: emptyEnvironment.shell`${nixRaw`pkgs.process-compose`}/bin/process-compose up -f ${configYml}`,
-    },
-    {}
-  );
+  return emptyEnvironment.shell`${nixRaw`pkgs.process-compose`}/bin/process-compose up -f ${configYml}`;
```

Comment:
Can we check that the description doesn't blow up here? I could imagine that the `configYml` being splatted in causes the description to be huge (low priority)

Reply:
Done in c09db3e3.